devTools/tsconfigs/tsconfig.base.json

```
@@ -1,23 +1,15 @@
 {
-    "$schema": "https://json.schemastore.org/tsconfig",
+    "extends": "@tsconfig/node22/tsconfig.json",
     "compilerOptions": {
         "module": "commonjs",
-        "strict": true,
-        "esModuleInterop": true,
         "resolveJsonModule": true,
         "forceConsistentCasingInFileNames": true,
-        "skipLibCheck": true,
         "composite": true,
 
         "declaration": true,
         "declarationMap": true,
 
-        // We start with https://github.com/tsconfig/bases/blob/master/bases/node14.json then drop:
-        //   "lib": ["es2020"],
-        //   "target": "es2020",
-
-        "lib": ["dom", "dom.iterable", "es2020", "es2021", "es2022", "es2023"],
-        "target": "es2019",
+        "lib": ["dom", "dom.iterable", "es2023"],
```

```suggestion
        "lib": ["dom", "dom.iterable", "es2023", "ESNext.Collection"],
```

should hopefully work :) (well, I haven't really extensively tried it, but I don't know why it wouldn't work)

---

Ahh, this might work for the new set intersection and friends. I'll try it in https://github.com/owid/owid-grapher/pull/4478